fiat-web/src/main/java/com/netflix/spinnaker/fiat/config/FiatConfig.java

```diff
@@ -123,11 +120,11 @@ public DefaultServiceAccountPredicateProvider defaultServiceAccountPredicateProv
 
   /**
    * This AuthenticatedRequestFilter pulls the email and accounts out of the Spring security context
-   * in order to enabling forwarding them to downstream components.
+   * in order to enable forwarding them to downstream components.
    */
   @Bean
-  FilterRegistrationBean authenticatedRequestFilter() {
-    val frb = new FilterRegistrationBean(new AuthenticatedRequestFilter(true));
+  FilterRegistrationBean<AuthenticatedRequestFilter> authenticatedRequestFilter() {
+    var frb = new FilterRegistrationBean<>(new AuthenticatedRequestFilter(true));
```

Comment:
MINOR thing... why the var vs. val?

Reply:
`val` is a Lombok extension while `var` is part of Java since 11 or so.